pkg/git/utils.go

```diff
@@ -33,6 +33,16 @@ func PrependIssueKeysToBranchName(issueKeys []string, branchName string) (string
 	return strings.Join(branchNameSubstrings, config.Options.IssueBranchSeparator), nil
 }
 
+// RemoveIssueKeysFromBranchName remove issue keys from branch name
+func RemoveIssueKeysFromBranchName(issueKeys []string, branchName string) string {
+	substrings := strings.Split(branchName, config.Options.IssueBranchSeparator)
```

Comment:
Usually, "sub" is written with "-". So what do you think to rename `substrings` to `subStrings`

Reply:
Good point, but when I was thinking about it I found that 'substring' in most of the cases is written without '-' indication  in software
https://www.w3schools.com/java/java_ref_string.asp

Reply:
True, also for go lang https://www.dotnetperls.com/substring-go